yolo:
    ros__parameters:
        input_topic: /yolo/input                                # topic with rgb, depth and odom
        output_topic: /yolo/output                              # topic with rgb, depth, mask, odom and bounding boxes
        create_mask: True                                       # node creates segmentation mask if True
        gpu_yolo_model: yolov8n-seg                             # yolov8 model for running on gpu
        cpu_yolo_model: yolov8n-seg                             # yolov8 model for running on cpu

keypoints_visualizer:
    ros__parameters:
        input_topic: /rtabmap/to_visualizer                     # input topic with images and keypoints
        output_topic: /rtabmap/keypoints/visualizer             # output_topic with keypoints images        
